crates/utils/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 pub mod errors;
 pub mod iter;
+pub mod json;
```

Comment:
```suggestion
pub mod format {
    pub mod json;
}
```

This makes it clear what `json` is from the module path. I think we can name `js_value_into_serde` and `js_value_from_serde` and reexport.

A better option is to have an extension trait that `JsValue` implements that provides these methods. That will also make the transition easier for wasm_bindgen users as all they need to do is add an import and remove the feature from Cargo.toml

Reply:
Got it. I will make it a trait instead.